### DISCUSSION TITLE
Setup generating error/ unable to install SecurityOnion

### DISCUSSION THREAD

**Discussion by pd98** [Security-Onion-Solutions/securityonion#6035] [Unsupported Versions]
SecurityOnion Version : 2.3.80
Install Source : ISO Image
Install Type : Standalone
Does so-status show all services running : Only Playbook shows error
Explain your issue : 

1. Tried installing SecurityOnion 2.3.80 on VSphere, with 16GB RAM, 4 cores, 500 GB storage, but the setup never completes and throws error. 
2. Not able to ssh into the machine to extract the so-setup log file even though the sshd service is running and the machine is pinging from the host. so-status shows all ok except Playbook which shows error. Tried to connect to the web interface after running so-allow but it shows connection refused.
3. Already tried reinstalling on a fresh instance, checked the checksum for the ISO image and also tried reinstalling with a newly downloaded ISO image.
4. Attaching screenshots of so-status and tail for so-setup log file since wasn't able to ssh to the machine.

![Screenshot_1](https://user-images.githubusercontent.com/33838706/139188559-c4aa8ed8-dae6-4b65-bce2-37b73d7f15b0.jpg)
![Screenshot_2](https://user-images.githubusercontent.com/33838706/139188573-6f399890-1a63-466a-985d-a9282059ea42.jpg)

Any help would be appreciated.

**Comment by dougburks** [comment]
Have you tried looking at sosetup.log for additional clues (beyond just the `tail -f sosetup.log`)?

Are there any network firewalls that could be interfering with your ssh connection?

**Reply by pd98** [reply]
Hi @dougburks ,

Yes I looked at the sosetup.log file but couldn't find any relevant information regarding the error. Most of the logs comprised of service startups and the total time. Only at the end I could see the error that I captured in the screenshot. 

And yes I checked, but no network firewalls were interfering with the web and ssh connection.

**Reply by dougburks** [reply]
Have you tried installing on a different vSphere host or a non-vSphere host?

**Reply by pd98** [reply]
@dougburks 
Yes tried installing on a different vSphere host, but it still ran into the the same problem.

**Reply by dougburks** [reply]
Have you tried on a non-vSphere host?

**Comment by abesinger** [comment]
@pd98 I just want to point out that SO definitely can run on Vsphere. I have a distributed setup running in Vsphere and had no problems installing it using the SO ISO image. (SO was at 2.3.60 when I did the install).

Not to dismiss your problem, just that it's something specific to your setup or how you're trying to install. It seems from what you said above that there are no errors in sosetup.log other than what you've shown.  You might also try checking system logs and running dmesg right after boot, see if there are any relevant errors there.

One other thought occurs to me, though I think it unlikely.  Configuring network interfaces on VMS for sniffing traffic is tricky (I'm not doing it; waiting to get dedicated hardware before running zeek and suricata).  Perhaps the SO install did something hinky with configuring the virtual NICs.
